src/modules/project/pages/projectView/views/projectMainBody/sections/Goals.tsx

```diff
@@ -141,3 +214,53 @@ export const Goals = () => {
     </>
   )
 }
+
+const SortableItem = ({
+  key,
+  goal,
+  editMode,
+  handleEditGoalModalOpen,
+}: {
+  key: string
+  goal: ProjectGoal
+  editMode: boolean
+  handleEditGoalModalOpen: (goal: ProjectGoal) => void
+}) => {
+  const { listeners, setNodeRef, transform, transition, attributes, isDragging } = useSortable({
+    id: goal.id.toString(),
+  })
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
```

Comment:
was this the recommended method for dnd-kit?

Reply:
Yes indeed